.github/workflows/slack.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     steps:
     - name: Get current star count
       run: |
-        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/meshery' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/mesheryctl-aliases' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/meshery/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/mesheryctl-aliases/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```

Comment:
Hey!, Thank you for the PR , please write this repo name in the place of `<new-repo>`

Reply:
Great feedback.